modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -688,6 +688,32 @@ AUI.add(
 					instances[id] = inputLocalizedInstance;
 				}
 
+				if (config.editor) {
+					Liferay.on('ckeditor:instanceReady', (event) => {
```

Comment:
This is a global handler.. it will easily leak if not destroyed.

Why not use `config.editor.getNativeEditor().on('instanceReady')`? That way you only register to the specific instance of editor and don't need to fire unnecessary global events.

Reply:
Yes, I hadn't thought about the global handler
Seems like a better idea, thanks

Reply:
I think the only problem I'm seeing with that, is that this will work the first time the page is loaded, and when resizing the window and re-creating the editor; the event handler is not called again ...  makes sense; but I still need to figure out how to make it work correctly